default_middlewares.go

```diff
@@ -0,0 +1,47 @@
+package fuego
+
+import (
+	"log/slog"
+	"net/http"
+)
+
+type LoggingConfig struct {
+	DisableRequest  bool
+	DisableResponse bool
+
+	RequestLogger  func(w http.ResponseWriter, r *http.Request)
+	ResponseLogger func(w http.ResponseWriter, r *http.Request)
+}
+
+func (l *LoggingConfig) Disabled() bool {
+	return l.DisableRequest && l.DisableResponse
+}
+
+var defaultLoggingConfig = LoggingConfig{
+	RequestLogger:  logRequest,
+	ResponseLogger: logResponse,
+}
+
+func defaultLoggingMiddleware(s *Server) func(http.Handler) http.Handler {
+	return func(next http.Handler) http.Handler {
+		return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			if !s.loggingConfig.DisableRequest {
+				s.loggingConfig.RequestLogger(w, r)
+			}
+
+			next.ServeHTTP(w, r)
+
+			if !s.loggingConfig.DisableResponse {
+				s.loggingConfig.ResponseLogger(w, r)
+			}
+		})
+	}
+}
+
+func logRequest(w http.ResponseWriter, r *http.Request) {
+	slog.Info("<- request")
+}
```

Comment:
Really? This seems a bit light! I would love to see at least the basics

- Method
- Path
- Timestamp

Reply:
Hey sorry this was just a placeholder for the draft pr while trying out the api for the logging... i plan on cleaning this up tommorow!

Reply:
Yeah hence my comment

>  Looks good. Now time for the part we're all very opinionated about what the logs like 😄

Reply:
Ah my bad the PR is indeed in draft, sorry.

Reply:
> Now time for the part we're all very opinionated about what the logs like 😄

Indeed. For the moment, I think, let's just use basic structured logging proposed by the standard library!

If we want to be opiniated on the logging, we'll have to implement a logging frontend (the "displayed", cf [this example](https://github.com/lmittmann/tint)) and keep the backend (the "generated log" structured)

Reply:
Something like this

```golang
slog.Info("<- request", "method", r.Method, "url", r.URL.String(), "timestamp", time.Now().Format(time.RFC3339))

```

Reply:
I think if we ever want to be opinionated, we should probably just shoot for a full fledged library in middlewares/.

also upvote for a requestid in the default as well